packages/js/components/package.json

```diff
@@ -132,18 +132,10 @@
 	"devDependencies": {
 		"@babel/core": "^7.23.5",
 		"@babel/runtime": "^7.23.5",
-		"@storybook/addon-actions": "6.5.17-alpha.0",
 		"@storybook/addon-console": "^1.2.3",
-		"@storybook/addon-controls": "6.5.17-alpha.0",
-		"@storybook/addon-docs": "6.5.17-alpha.0",
-		"@storybook/addon-knobs": "^6.4.0",
-		"@storybook/addon-links": "6.5.17-alpha.0",
-		"@storybook/addons": "6.5.17-alpha.0",
-		"@storybook/api": "6.5.17-alpha.0",
-		"@storybook/components": "6.5.17-alpha.0",
-		"@storybook/core-events": "6.5.17-alpha.0",
+		"@storybook/addon-docs": "7.6.19",
+		"@storybook/addon-links": "7.6.19",
 		"@storybook/react": "6.5.17-alpha.0",
```

Comment:
I had to keep `@storybook/react` to this version for now, as updating it caused unit tests to be run using React 18 for some reason and caused a bunch of random errors.
We can update this once we update React 18 repo wide.